samples/language/Namespaces.qs

```diff
@@ -2,19 +2,17 @@
 /// Namespaces
 ///
 /// # Description
-/// Every Q# file typically starts with a namespace. A namespace helps
+/// Every Q# file defines a namespace. A namespace helps
 /// you organize related functionality this is useful when you are writing
 /// libraries or reusable code.
-namespace MyQuantumApp {
+// The name of the file, minus the extension `.qs`, is the name of the namespace.
 
-    // The following `open` directive is used to import all types and callables declared in the
-    // Microsoft.Quantum.Diagnostics namespace.
-    open Microsoft.Quantum.Diagnostics;
+// The following `import` statement is used to import all types and callables declared in the
+// Std.Diagnostics namespace.
+import Std.Diagnostics.*;
 
-    @EntryPoint()
-    operation Main() : Result[] {
-        // `DumpMachine` is in the Microsoft.Quantum.Diagnostics namespace
-        DumpMachine();
-        return [];
-    }
-}
\ No newline at end of file
+operation Main() : Result[] {
```

Comment:
This should be a function. I'm surprised this lint doesn't cause failure in CI...

Reply:
Unfortunately, it seems that only samples in `samples.mjs` are checked for lint. See point 1 in https://github.com/microsoft/qsharp/issues/1709 filed earlier by me.

Reply:
Fixed, but also looked into why this is happening. It seems we set the default lint level to Allow for that lint: https://github.com/microsoft/qsharp/blob/main/compiler/qsc_linter/src/lints/hir.rs#L38

I didn't realize this, as I thought I was seeing this lint show up in VS Code. Maybe we should consider changing it to warn?

@Manvi-Agrawal  -- I saw that issue 😄  this PR introduces lint checks for all samples. See the functions [here](https://github.com/microsoft/qsharp/pull/1797/files#diff-16dc338c9c40d978fff7f2459d5ea48f1746c215d2608f635a83ae5ba0442c91R299) and [here](https://github.com/microsoft/qsharp/pull/1797/files#diff-e93cacfe927ccbf5b05cd6b97272ffc28432426179e07631686eed2a2cc1f6fdR153). It was a good call, thanks for filing that issue!

Reply:
Earlier, it was set to `Warn` by me and it was changed to `Allow` in https://github.com/microsoft/qsharp/pull/1627. I think it was done because katas might have Placeholders code as operations which doesnt fullfill operation requirements, but the actual solution is an operation.

Reply:
I see -- I missed the fact that we override it to warn everywhere else. I think the status quo is fine here, but I'll manually set the configuration for the `samples_test` crate to match how we have it elsewhere.

Reply:
Updating the lint config caught four more needless operations in samples! Updating now.

Reply:
Nice, do you think that the below test that executes samples in `samples.mjs` is relevant now? Given that we are now checking for lint and sample execution for all samples? Prolly getting rid of this test in this/other PR might be a good idea if its not needed.
https://github.com/microsoft/qsharp/blob/d1fb2a164a3ec4db73506beda3a70e097ddaacc9/npm/qsharp/test/basics.js#L350-#L370

Reply:
Worth some double checking, but I think you are probably right, the newer tests make this testing infra redundant. Good candidate for follow up PR!